src/config.js

```diff
@@ -289,11 +292,15 @@ global.config = {
     maxBuyPrice: 0.5,
     // buyByOwnOrders: true,
     buyOrderPriceMultiplicand: 0.5,
-
+    // buy energy, let make use of our credits
+    buyEnergy: {
+      enabled: true,
+      allowedSalesHistoryDeviation: 0.05,
+    },
     // buy power if we have more credits than config.market.minCredits
     buyPower: false,
-    // 300M credits
-    minCredits: 300000000,
+    // 300K credits
+    minCredits: 300000,
```

Comment:
I think we should have as minCredits the price of an access key, which is currently around 500M.
So we shouldn't reduce this, if you prefer you can change it in your local config.

What do you think?